libraries/plugins/market_history/market_history_plugin.cpp

```diff
@@ -410,7 +410,7 @@ void market_history_plugin::plugin_set_program_options(
    )
 {
    cli.add_options()
-         ("bucket-size", boost::program_options::value<string>()->default_value("[60,300,900,1800,3600,14400,86400]"),
+         ("bucket-size", boost::program_options::value<string>()->default_value("[15,60,300,3600,86400]"),
```

Comment:
change unrelated to readline

Reply:
Grrr. Git picked up a change from oxarbitrage. I'm not sure why. It is evidently something that happened when I rebased. I'll sort it out.